pkg/didcomm/protocol/exchange/exchange.go

```diff
@@ -86,29 +91,39 @@ func GenerateInviteWithKeyAndEndpoint(invite *Invitation) (string, error) {
 	return encodedExchangeInvitation(invite)
 }
 
+// Protocol for exchange protocol
+type Protocol struct {
+	prov transport.OutboundTransport
```

Comment:
prov is not meaningful here, outTransport makes more sense, see comment below

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/a31807a201229cd5ab417413af54914995b98b87..a26c159c381cbbb380ef97f616c8f9b9bddd476a#diff-4d3ca323f9b237cfa9940c12efa9b615R96).